tracpro/baseline/forms.py

```diff
@@ -0,0 +1,55 @@
+from django import forms
+
+from tracpro.polls.models import Poll
+from .models import BaselineTerm
+
+
+class BaselineTermForm(forms.ModelForm):
+    """
+    Form for Baseline Term
+    """
```

Comment:
I've made this adjustment, please take a look at static\js\baseline.js and test it out.